src/main/java/net/sacredlabyrinth/phaed/simpleclans/listeners/LandProtection.java

```diff
@@ -240,6 +241,12 @@ public void registerCreateLandEvent(ProtectionProvider provider, @Nullable Class
         }, plugin, true);
     }
 
+    private boolean isIgnored(ProtectionManager.Action action, Block block) {
```

Comment:
Move to the SettingsManager, make this method public, then use a ternary operator.

Reply:
> Move to the SettingsManager, make this method public, then use a ternary operator.

done